.github/workflows/govcloud.yml

```diff
@@ -126,3 +126,9 @@ jobs:
           name: accessibility-test-report
           path: /home/runner/work/funding-service-design-frontend/funding-service-design-frontend/axe_reports/*.html
           retention-days: 5
+      - name: ZAP Scan
+        uses: zaproxy/action-full-scan@v0.3.0
+        with:
+          target: ${{ secrets.FORMS_SERVICE_HOST }}
+          allow_issue_writing: False
+          
```

Comment:
It seems the ZAP action by default doesn't exit with an error (even if high security alerts) unless you set it to with this flag - so this should be added at the bottom `fail_action: true`